src/TimeoutError.ts

```diff
@@ -2,9 +2,9 @@ export default class TimeoutError extends Error {
   public timeout: number;
   public statusCode: number;
 
-  constructor(message: string, timeout?: number, statusCode?: number) {
+  constructor(message: string, timeout: number, statusCode: number) {
```

Comment:
What do you think about moving the default status code for timeout errors into the TimeoutError class like this:

<pre>
constructor(message: string, timeout: number, statusCode: number = 408)
</pre>

I'm thinking it makes more sense for the TimeoutError class to know an appropriate default status code rather than its client. You can then make the statusCode passed to timeoutLink optional.

Reply:
> You can then make the statusCode passed to timeoutLink optional.

The TimeoutLink's statusCode parameter is already optional because it uses an initializer. I can create a TimeoutLink like `new TimeoutLink(1000)` without a TS Error.

I was trying to keep the default values in the same place, but I'm happy to move it into the error if you'd prefer 👍

Reply:
Well, the default value was originally stored in the TimeoutError file but it looks like you moved it to the timeoutLink file. So yeah, if you can move it back that'd be great. I think the rest looks good.

Reply:
Done 👍 

Sorry, I was unclear - I initially moved the statusCode default so that it was co-located with the timeout default. I agree with you though, it makes sense for the TimeoutError to hold the statusCode default and the link to hold the timeout default.